default.json5

```
@@ -77,22 +77,64 @@
       ],
     },
     {
-      groupName: "all major dependencies",
-      groupSlug: "all-major",
+      matchManagers: [
+        "gradle",
+      ],
+      groupName: "gradle all major dependencies",
+      groupSlug: "all-major-gradle",
```

please remove the `matchManagers` from the upper rules
```suggestion
      groupName: "all major dependencies",
      groupSlug: "all-major",
```